apps/aestratum/priv/Payout.aes

```diff
@@ -0,0 +1,11 @@
+contract Payout =
+
+  public function payout(xs : list((address, int))) : int =
+    payout'(xs, 0)
+
+  function payout'(xs : list((address, int)), total : int) : int =
+    switch(xs)
+      [] => total
+      (address, tokens) :: xs' =>
+        Chain.spend(address, tokens)
```

Comment:
What is impact if tokens zero?

What is impact if tokens negative?

Reply:
Negative tokens will result in a crash -> out of gas.

Reply:
Zero case is prevented here: https://github.com/aeternity/aeternity/blob/dev-stratum/apps/aestratum/src/aestratum_reward.erl#L88

And negative case here: https://github.com/aeternity/aeternity/blob/dev-stratum/apps/aestratum/src/aestratum_reward.erl#L85